src/vm/loaderallocator.cpp

```diff
@@ -1032,7 +1032,7 @@ void LoaderAllocator::ActivateManagedTracking()
 #define COLLECTIBLE_HIGH_FREQUENCY_HEAP_SIZE       (3 * GetOsPageSize())
 #define COLLECTIBLE_STUB_HEAP_SIZE                 GetOsPageSize()
 #define COLLECTIBLE_CODEHEAP_SIZE                  (7 * GetOsPageSize())
-#define COLLECTIBLE_VIRTUALSTUBDISPATCH_HEAP_SPACE (5 * GetOsPageSize())
+#define COLLECTIBLE_VIRTUALSTUBDISPATCH_HEAP_SPACE (6 * GetOsPageSize())
```

Comment:
This is carefully tuned to sum up to 16 pages. It will sum up to 17 pages now and we will be wasting 60kB of address space per collectible loader allocator on Windows.

I do think we need to preallocate anything space for the new stubs in collectible contexts. It is unlikely that R2R images will get loaded in the collectible context.